_data/sw_servers.yml

```diff
@@ -40,6 +40,26 @@
           starttls:
           userhost-in-names:
           webirc:
+    - name: ChatIRCd
+      # dev: Ben / MrC
+      # ref: https://bitbucket.org/chatlounge/chatircd/commits/all
+      link: http://www.chatlounge.net/software/
```

Comment:
404

Reply:
@MrBenC can you file a PR to update the ChatIRCd website link some time?